Remove the sysctl.h include from makehfs.c. Every sysctl() call site is
inside #if !LINUX, and the header does not exist outside Linux.

diff --git a/newfs_hfs.tproj/makehfs.c b/newfs_hfs.tproj/makehfs.c
index c1aeca0..796e7a4 100644
--- a/newfs_hfs.tproj/makehfs.c
+++ b/newfs_hfs.tproj/makehfs.c
@@ -38,7 +38,6 @@
 #endif
 #include <sys/errno.h>
 #include <sys/stat.h>
-#include <linux/sysctl.h>
 #if !LINUX
 #include <sys/vmmeter.h>
 #endif
